01-js-intro-and-jquery/10/js/scripts.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready(function() {
+	
+	var alias = '';
+	var name  = '';
+	var nameLength = '';
+	var lastChange = '';
+	
+	var focusAlias = function() {	
+		$('.alias').focus(); 
+	};
+
+	var highlightName = function() {
+		name = $('#response').html();
```

Comment:
This will return an HTML chunk (therefore your ``substring`` output could return undesired characters and you have to specify a hardcoded starting index to avoid including the tag), which is incorrect. You could use jQuery's ``text()`` or JS ``textContent``. Also, you could process ``name`` variable in one line:
```
name = $('#response').html().substring(8);
```